src/api/firebase.ts

```diff
@@ -301,3 +301,44 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export function comparePurchaseUrgency(item1: ListItem, item2: ListItem) {
```

Comment:
You might be able to update this function's return type to be a bit more explicit

```
export function comparePurchaseUrgency(item1: ListItem, item2: ListItem): -1 | 0 | 1 {
```